lib/api/noirlabReleases.js

```diff
@@ -51,9 +53,10 @@ export function useRelease(site, entry) {
 
 export function useReleases(site, entries = []) {
   const locale = getLocaleString(site);
-  const releaseEntries = entries.filter((entry) => entry.pressReleaseId);
-
-  const getRel = (url) => fetch(url).then((res) => res.json());
+  const releaseEntries = useMemo(
```

Comment:
this had to be memo-ified because it was re-rendering endlessly. Not sure what caused it or why it turned up now, chalked it up to React stuff that happens when you move project structures around